testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/util/DeltaLakeTestUtils.java

```diff
@@ -125,4 +125,14 @@ public static void removeS3Directory(AmazonS3 s3, String bucketName, String dire
         }
         while (listing.isTruncated());
     }
+
+    public static String getLocationForSchema(String externalLocation, String schemaName)
+    {
+        return format("%s/%s", externalLocation, schemaName);
+    }
+
+    public static String getLocationForTable(String externalLocation, String schemaName, String tableName)
+    {
+        return format("%s/%s/%s", externalLocation, schemaName, tableName);
+    }
```

Comment:
These methods are very simple and the implementation is just a `format` method. I would inline them.